server/server-kv.c

```diff
@@ -89,6 +140,9 @@ server_main(int sockfd, char* thread_number)
                 addr_size = sizeof client_addr;
                 clientfd = accept(sockfd, (struct sockaddr *)&client_addr, &addr_size);
                 //sh_print_client_ip(client_addr);
+		#ifdef DEBUG
```

Comment:
It's even easier, actually. Merge in the branch I linked and use the function `kvprintf()` which will only print if the verbose flag is set